tests/functional_tests/iterator_tests/test_toy_connectivity.py

```diff
@@ -110,15 +110,15 @@ def sum_edges_to_vertices(in_edges):
     )
 
 
-def test_sum_edges_to_vertices(fencil_processor):
-    fencil_processor, validate = fencil_processor
+def test_sum_edges_to_vertices(fencil_processor_no_gtfn_exec):
+    fencil_processor_no_gtfn_exec, validate = fencil_processor_no_gtfn_exec
```

Comment:
```suggestion
    fencil_processor, validate = fencil_processor_no_gtfn_exec
```
As mentioned in #869 I didn't see the point of doing this. Activating the tests is much easier and the rest of the test remains more readable.

Reply:
The only reason was that my find/replace was simpler. I can change it...